packages/editor-ui/src/modules/nodeTypes.ts

```diff
@@ -0,0 +1,87 @@
+import Vue from 'vue';
+import { ActionContext, Module } from 'vuex';
+import type { INodeTypeDescription } from 'n8n-workflow';
+
+import type { IRootState, INodeTypesState } from '../Interface';
+import { DEFAULT_NODETYPE_VERSION } from '@/constants';
+import { getNodeTypes } from '@/api/nodeTypes';
+
+const module: Module<INodeTypesState, IRootState> = {
+	namespaced: true,
+	state: {
+		nodeTypes: {},
+	},
+	getters: {
+		allNodeTypes: (state): INodeTypeDescription[] => {
+			return Object.values(state.nodeTypes);
```

Comment:
does sort order matter in some places? here we might lose sort order that was set by api before

Reply:
Not sure about this? In `master` when we have `const newNodesState = [...oldNodesToKeep, ...nodeTypes]` in `updateNodeTypes` and then `state.workflow.nodes` in `allNodes`.

Reply:
ya I guess in nodes panel we sort them anyways..